overrides/config/AppliedEnergistics2/AppliedEnergistics2.cfg

```diff
@@ -0,0 +1,382 @@
+# Configuration file
+
+automation {
+    I:formationPlaneEntityLimit=128
+}
+
+
+battery {
+    I:chargedStaff=8000
+    I:colorApplicator=20000
+    I:entropyManipulator=200000
+    I:matterCannon=200000
+    I:portableCell=20000
+    I:wirelessTerminal=1600000
+}
+
+
+client {
+    # Possible Values: AE, EU, WA, RF, MK
+    S:PowerUnit=AE
+
+    # Possible Values: AUTOSEARCH, MANUAL_SEARCH, NEI_AUTOSEARCH, NEI_MANUAL_SEARCH
+    S:SEARCH_MODE=AUTOSEARCH
+
+    # Possible Values: YES, NO, UNDECIDED
+    S:SEARCH_TOOLTIPS=YES
+
+    # Possible Values: TALL, FULL, SMALL
+    S:TERMINAL_STYLE=TALL
+
+    # Controls buttons on Crafting Screen : Capped at 9
+    I:craftAmtButton1=1
+
+    # Controls buttons on Crafting Screen : Capped at 99
+    I:craftAmtButton2=10
+
+    # Controls buttons on Crafting Screen : Capped at 999
+    I:craftAmtButton3=100
+
+    # Controls buttons on Crafting Screen : Capped at 9999
+    I:craftAmtButton4=1000
+    B:disableColoredCableRecipesInNEI=true
+    B:enableEffects=true
+
+    # Controls buttons on Level Emitter Screen : Capped at 9
+    I:levelAmtButton1=1
+
+    # Controls buttons on Level Emitter Screen : Capped at 99
+    I:levelAmtButton2=10
+
+    # Controls buttons on Level Emitter Screen : Capped at 999
+    I:levelAmtButton3=100
+
+    # Controls buttons on Level Emitter Screen : Capped at 9999
+    I:levelAmtButton4=1000
+
+    # Controls buttons on Priority Screen : Capped at 9
+    I:priorityAmtButton1=1
+
+    # Controls buttons on Priority Screen : Capped at 99
+    I:priorityAmtButton2=10
+
+    # Controls buttons on Priority Screen : Capped at 999
+    I:priorityAmtButton3=100
+
+    # Controls buttons on Priority Screen : Capped at 9999
+    I:priorityAmtButton4=1000
+    B:useColoredCraftingStatus=true
+    B:useTerminalUseLargeFont=false
+}
+
+
+condenser {
+    I:MatterBalls=256
+    I:Singularity=256000
+}
+
+
+craftingcpu {
+    I:craftingCalculationTimePerTick=5
+}
+
+
+features {
+
+    world {
+        B:CertusQuartzWorldGen=true
+        B:ChestLoot=true
+        B:DecorativeLights=true
+        B:DecorativeQuartzBlocks=true
+        B:Flour=true
+        B:GrindStone=true
+        B:Inscriber=true
+        B:MeteoriteWorldGen=true
+        B:SkyStoneChests=true
+        B:SpawnPressesInMeteorites=true
+        B:TinyTNT=true
+        B:VillagerTrading=true
```

Comment:
Do you know which trades are added and if they're balanced? For some Pam's Addon we removed some trades before I think.